src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3994,6 +3994,7 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
     std::map<std::string, std::tuple<int64_t, bool, Location>> goto_name2id;
     int64_t gotoids;
     std::vector<ASR::symbol_t*> do_loop_variables;
+    std::set<std::string> imported_functions;
```

Comment:
What is the purpose of this variable? Let's document it as a comment.

Is it to keep track which functions were imported? Don't we need to also track which modules they were imported from?